iModelCore/GeomLibs/geom/test/src/checkers.cpp

```diff
@@ -1839,14 +1839,12 @@ void Check::TearDown()
     }
 
 // verify geometry round trip through both JSON and FlatBuffer
-bool Check::NearRoundTrip(IGeometryCR g, double tolerance, char const* pString)
+bool Check::NearRoundTrip(IGeometryCR g, double tolerance, WCharCP pString)
     {
+    std::wstring ws(pString);
     std::string myString;
-    if (pString)
-        {
-        myString.append(pString);
-        myString.append(": ");
-        }
+    std::transform(ws.begin(), ws.end(), std::back_inserter(myString), [](wchar_t c){ return (char)c; });

```

Comment:
This casting of wchar_t to char is bad practice, because it won't always work.

This could be the tip of the iceberg, but the cleanest change would be to propagate the WCharCP arg to the other Check::XXX methods called by `Check::NearRoundTrip`. In other words, change those called methods' signatures too.

If you don't want to do that, the conventional way to turn `pString` into a `char` string is using `BeStringUtilities::WCharToCurrentLocaleChar`.  See WString.h for example usage.

WString("B-spline surface: ", BentleyCharEncoding::Locale).c_str()